internal/internal_workflow_testsuite.go

```diff
@@ -534,6 +534,38 @@ func (env *testWorkflowEnvironmentImpl) executeActivity(
 	}
 }
 
+func (env *testWorkflowEnvironmentImpl) executeLocalActivityParams(
```

Comment:
what is the difference between this and executeLocalActivity ? Why do you need to return concrete type here ?Given that this is unit test, can you type case inside of your test inside of creating this additional method which does the same thing ?